src/runtime/Story.svelte

```diff
@@ -47,20 +38,20 @@
      * @deprecrated
      * Use `tags={['autodocs']}` instead.
      */
-    autodocs?: never;
+    autodocs?: boolean;
     /**
      * @deprecated
      * Use `parameters={{ docs: { source: { code: "..." } } }}` instead.
      */
-    source?: never;
+    source?: boolean | string;
```

Comment:
These aren't correct, are they? We don't support `autodocs` and `source` attributes in the new `Story`, only in `LegacyStory`? I can't remember why we had them as type `never` instead of just removing them completely though. the "deprecated" tag is misleading, since these are actually removed, not just deprecated.

Reply:
Yes, they're removed from the modern `Story` component.

The opt-in legacy support option handles them in `LegacyStory` with pre-transform plugin.

The reason I brought them back was to prevent users from having waterfall of type errors caused by usage of these props. It would occur while upgrading this addon in their existing project(s) to the next major version.

Hmm... Now given the second thought reading my reply, I should've put these type changes inside the `LegacyStory` instead. That sounds reasonable to you?

Reply:
hmm. 

> The reason I brought them back was to prevent users being flooded with type errors caused by usage of these props. It would occur while upgrading this addon in their existing project(s) to the next major version.

This seems reasonable to me.

> Hmm... Now given the second thought reading my reply, I should've put these type changes inside the LegacyStory instead. That sounds reasonable to you?

And this is also what would make sense to me - but doing this means that users will still get flooded with type errors, right? because no user is explicitly importing LegacyStory? I'm conflicted here. We could document that to get rid of these type errors, users need to do a search and replace for `Story -> LegacyStory` in their code base? But I'm not sure that even works at runtime, does it?

I'll defer to your best judgement here, I trust you.

Reply:
> And this is also what would make sense to me - but doing this means that users will still get flooded with type errors, right? **because no user is explicitly importing LegacyStory**? I'm conflicted here. We could document that to get rid of these type errors, users need to do a search and replace for `Story -> LegacyStory` in their code base? But I'm not sure that even works at runtime, does it?

Hm, I'm trying to get how you see that they could be flooded by those type errors? I'm guessing there's a confusion between `Story` and `LegacyStory` _(which is exported and aliased as `Story` for legacy compatibility)_.

Below is legacy syntax:

```svelte
<script>
  import { Story } from '@storybook/addon-svelte-csf';
  //       👆 This one is the `LegacyStory` now, so its old props are left untouched, except for the deprecation remarks.
</script>
```

Reply:
> which is exported and aliased as Story for legacy compatibility

Sorry, I totally forgot about that part. 😵‍💫